src/adapters/equirectangular-tiles/index.js

```diff
@@ -394,7 +384,7 @@ export default class EquirectangularTilesAdapter extends AbstractAdapter {
     const panorama = this.psv.config.panorama;
     const url = panorama.tileUrl(tile.col, tile.row);
 
-    return new Promise((resolve, reject) => this.loader.load(url, resolve, undefined, reject))
+    return this.psv.textureLoader.loadImage(url)
```

Comment:
I cannot accept this change. When developping this adapter I had performances issues with the FileLoader which were resolved by using the ImageLoader.

Reply:
The performance issue is caused by the double step loading in PSV TextureLoader (blob then image). Because the Tiles adapter loads a lot of images it has to be very lightweight.

Reply:
ImageLoader also has a "setRequestHeader" method, it is just a matter of duplicating the code.

Reply:
Hi I have tried this. The THREE.ImageLoader doesn't work with a setRequetsHeader because it just uses img.src = "". So if you facing performances issues I suggest not to implement the header for the tiles. So you can remove the code from the index.js of the adapter.